cognitedata/reveal
# feat: add clustering capabilities to 360 image collections

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-782
https://cognitedata.atlassian.net/browse/REV-784

## Description :pencil:


Adds clustering icons and logic for 360 image icons. 


## Test instructions :information_source:



Load a 360 image data set and notice that icons get clustered when zooming in / out.


## Checklist :ballot_box_with_check:
